examples/mega-form/src/useAtomSlice.ts

```diff
@@ -6,7 +6,11 @@ import { splitAtom } from 'jotai/utils'
 const useAtomSlice = <Item>(arrAtom: PrimitiveAtom<Item[]>) => {
   const [atoms, remove] = useAtom(useMemo(() => splitAtom(arrAtom), [arrAtom]))
   return useMemo(
-    () => atoms.map((itemAtom) => [itemAtom, () => remove(itemAtom)] as const),
+    () =>
+      atoms.map(
+        (itemAtom) =>
+          [itemAtom, () => remove({ type: 'remove', atom: itemAtom })] as const,
```

Comment:
Nice catch! It was an old style.

Let's rename `remove` to `dispatch`.
```suggestion
          [itemAtom, () => dispatch({ type: 'remove', atom: itemAtom })] as const,
```